config/config.go

```diff
@@ -98,7 +102,7 @@ func init() {
 	cfg.SetDefault("etcd.data_dir", "/var/lib/skydive/etcd")
 	cfg.SetDefault("etcd.embedded", true)
 	cfg.SetDefault("etcd.name", host)
-	cfg.SetDefault("etcd.listen", "127.0.0.1:12379")
+	cfg.SetDefault("etcd.listen", fmt.Sprintf("127.0.0.1:%d", etcdDefaultPort))
```

Comment:
nit: string concatenation won't be sufficient?

Reply:
that's string and integer, so will have to convert it anyway